### ISSUE TITLE
Realm Swift 0.96.0-1: ignoredProperties no longer working

### ISSUE THREAD

**Issue by mlusas** [realm/realm-cocoa#2732]
After updating to 0.96+, my ignoredProperties are no longer ignored, and are now included in the Realm database.

**After updating, I now receive an error of:**
Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[_NSZeroData setObject:]: unrecognized selector sent to instance <instanceID>

The instance refers to the NSData var I'm passing into my model.  Has anyone been able to fix this, and get ignored properties working again?  Here is my model for that object.  The commented out code are variations I tried to due documented variations in ignoring properties; none of these worked.

**Realm Model:**
```
class ImageFile: Object {
    dynamic var imageID:String = ""
    dynamic var imageFileID:String = ""
    dynamic var imageType:String = ""
    dynamic var selectedImageFile:Bool = false
    dynamic var folder:String = ""
    dynamic var filename:String = ""
    dynamic var scale:Int = 0
    dynamic var height:Float = 0
    dynamic var width:Float = 0
    dynamic var created:Int = 0
    dynamic var updated:Int = 0
    dynamic var expires:Int = 0
//    dynamic var imageData:NSData? = NSData()
    var imageData:NSData? = NSData()
    
    override static func primaryKey() -> String {
        return "imageFileID"
    }
    
    override static func indexedProperties() -> [String] {
        return [
            "imageID",
            "imageType",
            "selectedImageFile",
            "scale"
        ]
    }
    
//    override static func ignoredProperties() -> [String] {
//        return ["imageData"]
//    }
    
    override class func ignoredProperties() -> [String] {
        return ["imageData"]
    }
}
```

**Comment by kishikawakatsumi** [comment]
@mlusas Thank you for reporting this. I've reproduced the same issue. It seems a bug. We'll have a fix out shortly.

**Comment by mlusas** [comment]
@kishikawakatsumi Wonderful, thank you for the quick response.

**Comment by kishikawakatsumi** [comment]
Fixed #2734
